src/authorization/AuthorizationsDefinition.ts

```diff
@@ -487,7 +487,7 @@ export const AUTHORIZATION_DEFINITION: AuthorizationDefinition = {
           'chargeAlternativePhase', 'chargeOptionPower', 'chargeOptionPhase', 'image', 'chargeOptionPhaseAmp', 'chargeAlternativePhaseAmp'
         ]
       },
-      { resource: Entity.CAR, action: [Action.CREATE, Action.UPDATE, Action.DELETE] },
+      { resource: Entity.CAR, action: [Action.CREATE, Action.CREATE_POOL_CAR, Action.UPDATE, Action.DELETE] },
```

Comment:
Action.CREATE_POOL_CAR should be a CREATE + do not provide the option Pool Car in the UI + Assert to not have the value Pool Car in the backend.